src/components/molecules/organization-result/organization-result.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Avatar from 'components/atoms/avatar';
+
+import {
+  OrganizationResultContainer,
+  AvatarContainer,
+  DetailsContainer,
+  StatsContainer,
+  Row,
+  NameDisplay,
+  UsernameDisplay,
+} from './organization-result.styles';
+
+const OrganizationResult = ({
+  avatarUrl,
+  description,
+  email,
+  location,
+  login,
+  name,
+  url,
+}) => (
+  <OrganizationResultContainer href={url} target="_blank">
+    <AvatarContainer>
+      <Avatar atl={login} src={avatarUrl} />
+    </AvatarContainer>
+    <DetailsContainer>
+      <Row>
+        <NameDisplay>{name}</NameDisplay>
+        <UsernameDisplay>{login}</UsernameDisplay>
+      </Row>
+      <Row>
+        <StatsContainer>{description}</StatsContainer>
+      </Row>
+      <Row>
+        <StatsContainer>{location || 'Location Unavailable'}</StatsContainer>
+        <StatsContainer>{email || 'Email Unavailable'}</StatsContainer>
+      </Row>
+    </DetailsContainer>
+  </OrganizationResultContainer>
```

Comment:
These components and the backing styles are virtually identical to user-result. There's some other cleanup here and reusable components we could make and make this molecule a lot nicer and more visually appealing to look at. I think ideally there's be a `SearchResult` base component that this plugged into with specifics about an org and ditto for the user component.